ext/java/org/msgpack/jruby/Packer.java

```diff
@@ -30,20 +35,139 @@ public IRubyObject allocate(Ruby runtime, RubyClass type) {
     }
   }
 
+  static class ExtRegistry {
```

Comment:
Call this `ExtensionRegistry`. Short names are hard to understand; long, descriptive names don't cost anything.